index.html

```diff
@@ -86,17 +85,17 @@ <h5>Social Icons</h5>
             <a href="https://linkedin.com"><i class="fa fa-linkedin"></i></a>
             <a href="https://pinterest.com"><i class="fa fa-pinterest-square"></i></a>
           </div>
-          <hr>
-          <div class="card-body invisible-text">
+          <hr class="invisible-text">
+          <div class="card-body invisible-text" dir="rtl">
```

Comment:
Let's change the order of inner elements in DOM, instead of relying on [dir="rtl"](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir).